Source/Core/Core/ConfigManager.cpp

```diff
@@ -146,6 +146,7 @@ void SConfig::SaveGeneralSettings(IniFile& ini)
   general->Set("GDBSocket", gdb_socket);
 #endif
   general->Set("GDBPort", iGDBPort);
+  general->Set("TCPGeckoPort", iTCPGeckoPort);
```

Comment:
We're currently migrating to a new config system. Please add any new options to it (MainSettings.h/MainSettings.cpp) instead of adding more things to ConfigManager.